PR-template.txt

```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+[ ] Each commit in the pull request should have a meaningful subject line and body.
+[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
+[ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
+[ ] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
+Below are the contribution guidelines:
+https://github.com/apache/camel/blob/master/CONTRIBUTING.md
```

Comment:
They are here https://camel.apache.org/camel-k/latest/developers.html

Reply:
Thank you for the feedback! How does the below-proposed changes look?

[ ] Make sure there is a [Github issue](https://github.com/apache/camel-k/issues) filed for the change (before you start working on an issue).  One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-k/latest/developers.html